ci-operator/config/redhat-appstudio/build-service/redhat-appstudio-build-service-main.yaml

```diff
@@ -35,6 +35,8 @@ tests:
         export GITHUB_USER=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/github-user)
         export GITHUB_TOKEN=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/github-token)
         export QUAY_TOKEN=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/quay-token)
+        export QUAY_OAUTH_TOKEN_RELEASE_BUILD=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/quay-oauth-token-release-build)
+        export QUAY_OAUTH_TOKEN_RELEASE_RELEASE=$(cat /usr/local/ci-secrets/redhat-appstudio-qe/quay-oauth-token-release-release)
```

Comment:
Afaik there's no need to add them to build-service config. This job doesn't run any "release-specific" test

Reply:
lt's just a naming, 
you are right will rename it according to repo 
Talked to Meer and the naming will be as below and the tokens will be in release-service e2e-job :
QUAY_OAUTH_TOKEN_RELEASE_SOURCE
QUAY_OAUTH_TOKEN_RELEASE_DESTINATION